src/inner-slider.js

```diff
@@ -97,6 +97,15 @@ export class InnerSlider extends React.Component {
         slide.onblur = this.props.pauseOnFocus ? this.onSlideBlur : null;
       }
     );
+
+    if (this.props.touchMove) {
+      // swipeMove calls preventDefault which only has an effect on non-passive events
+      // https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener
+      this.list.addEventListener("touchmove", this.swipeMove, {
+        passive: false
```

Comment:
good to check if `animating` is true, if its true you can have passive true which will improve performance
as you are promising the handler won't call `preventDefault` to disable scrolling.